opencog/atomspace/GenericTruthValue.cc

```diff
@@ -0,0 +1,162 @@
+/*
+ * GenericTruthValue.cc
+ *
+ * Copyright (C) 2015 OpenCog Foundation
+ *
+ * Author: Leung Man Hin <https://github.com/leungmanhin>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the exceptions
+ * at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include <math.h>
+#include "GenericTruthValue.h"
+
+using namespace opencog;
+
+#define KKK 800.0f
+#define CVAL  0.2f
+
+GenericTruthValue::GenericTruthValue(count_t pe, count_t te,
+                                     strength_t f, strength_t fs,
+                                     confidence_t c, entropy_t e)
+{
+    positiveEvidence = pe;
+    totalEvidence = te;
```

Comment:
are these counts? or averages? I would like to see them called "count" if they are meant to integer-like .. e.g. positive EvidenceCount or just positiveCount.